packages/components/src/theme/ComposerTheme/Components/Symbols.ts

```diff
@@ -0,0 +1,17 @@
+import { ComponentStyleConfig } from '@chakra-ui/react';
+
+export const Symbols: ComponentStyleConfig = {
+  baseStyle: {
+    textStyle: 'text-body-meta-regular',
```

Comment:
I believe this is not adding the proper styling. Could you apply textStyles like we use to do for other components?

Reply:
Fixed